packages/query-core/src/tests/queryClient.test.tsx

```diff
@@ -433,6 +433,17 @@ describe('queryClient', () => {
       ).resolves.toEqual('bar')
     })
 
+    test('should return the cached query data if the query is found and cached query data is falsey', async () => {
+      const key = queryKey()
+      const queryFn = () => Promise.resolve('data')
```

Comment:
good fix, but the returned data here isn't falsy?

Reply:
Thanks. My intention was to test the case where the cache data is falsey and the new data is not falsey.

Reply:
0557237b22bf09cb9b4d0e5b15d0efe1b997dce1
make the test case more intuitive, and I have fixed all the returned data covered by the test case to be falsy values.
If there are any parts that need improvement, please let me know.